manager/Makefile

```diff
@@ -22,7 +22,7 @@ manager:
 # Run against the configured Kubernetes cluster in ~/.kube/config
 .PHONY: run
 run:
-	go run -ldflags "-X main.gitCommit=$(GIT_COMMIT)" ./main.go --enable-all-controllers --metrics-bind-addr=0
+	PRETTY_LOGGING=true LOGGING_VERBOSITY=-1 go run -ldflags "-X main.gitCommit=$(GIT_COMMIT)" ./main.go --enable-all-controllers --metrics-bind-addr=0
```

Comment:
Why adding them here? There are other env vars as well, I believe all should be set in one place.
Also, we need a default for these vars if they are not set.

Reply:
The comments were implemented. Thanks